vignettes/faq.Rmd

```diff
@@ -0,0 +1,77 @@
+---
+title: "FAQ"
+output: 
+  bookdown::html_document2:
+    fig_caption: yes
+    code_folding: show
+    number_sections: false
+pkgdown:
+  as_is: true
+# csl: https://raw.githubusercontent.com/citation-style-language/styles/master/apa-numeric-superscript-brackets.csl
+link-citations: true
+vignette: >
+  %\VignetteIndexEntry{FAQ}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+bibliography: references.bib
+---
+
+```{r include=FALSE}
+library(epidist)
```

Comment:
if you hide thsi how are peope reading it going to reproduce? What are you thinking the workflow will look like here?

Reply:
Yeah good point / something I'd thought a little about but think there are better solutions. It's pretty typical that in docs like this they aren't reproducible. But I think it should be reproducible. Maybe have it in a > section (code folded). Don't want to distract from the main body. What do you think?

Reply:
I like the code folding approach. Nice one